### ISSUE TITLE
`java.lang.AbstractMethodError` with Mojmap and NeoForge

### ISSUE THREAD

**Issue by moritz-htk** [architectury/architectury-loom#211]
~~Since I have been using Loom version 1.6.396~~ I get this error when I try to start NeoForge. When I start Fabric, everything works as usual.
I use Minecraft 1.20.6, ~~Yarn Mappings 1.20.6+build.1~~, Fabric Loader 0.15.11, Fabric API 0.98.0+1.20.6 and NeoForge 20.6.48-beta
```
Exception in thread "main" java.lang.AbstractMethodError: Receiver class cpw.mods.bootstraplauncher.BootstrapLauncher$PackageTracker does not define or inherit an implementation of the resolved method 'abstract boolean test(java.lang.String, java.lang.String)' of interface cpw.mods.niofs.union.UnionPathFilter.
	at cpw.mods.securejarhandler/cpw.mods.niofs.union.UnionFileSystem.testFilter(UnionFileSystem.java:463)
	at cpw.mods.securejarhandler/cpw.mods.niofs.union.UnionFileSystem.findFirstFiltered(UnionFileSystem.java:307)
	at cpw.mods.securejarhandler/cpw.mods.niofs.union.UnionFileSystem.checkAccess(UnionFileSystem.java:335)
	at cpw.mods.securejarhandler/cpw.mods.niofs.union.UnionFileSystemProvider.checkAccess(UnionFileSystemProvider.java:202)
	at java.base/java.nio.file.spi.FileSystemProvider.exists(FileSystemProvider.java:1202)
	at java.base/java.nio.file.Files.exists(Files.java:2514)
	at cpw.mods.securejarhandler/cpw.mods.jarhandling.impl.JarContentsImpl.lambda$findFile$6(JarContentsImpl.java:148)
	at java.base/java.util.Optional.filter(Optional.java:218)
	at cpw.mods.securejarhandler/cpw.mods.jarhandling.impl.JarContentsImpl.findFile(JarContentsImpl.java:148)
	at cpw.mods.securejarhandler/cpw.mods.jarhandling.JarMetadata.from(JarMetadata.java:57)
	at cpw.mods.securejarhandler/cpw.mods.jarhandling.SecureJar.from(SecureJar.java:43)
	at cpw.mods.bootstraplauncher.BootstrapLauncher.lambda$run$6(BootstrapLauncher.java:132)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
	at java.base/java.util.stream.SortedOps$SizedRefSortingSink.end(SortedOps.java:357)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:510)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at cpw.mods.bootstraplauncher.BootstrapLauncher.run(BootstrapLauncher.java:122)
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:65)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)
	at dev.architectury.transformer.TransformerRuntime.main(TransformerRuntime.java:219)
```

**Comment by Luligabi1** [comment]
I have the same issue, although I'm using mojmap, so it's likely not related.

I started having this issue when I upgraded Neoforge from `20.5.13-beta` to `20.5.50-beta` and can confirm downgrading does stop it.

**Comment by Fuzss** [comment]
This happens when updating NeoForge to `20.5.14-beta` or newer. The version of ArchLoom does not matter, was able to reproduce this with any of the `1.6.X` releases.

**Comment by Jab125** [comment]
> This happens when updating NeoForge to `20.5.14-beta` or newer. The version of ArchLoom does not matter, was able to reproduce this with any of the `1.6.X` releases.

With Yarn or Mojmap?

**Comment by Fuzss** [comment]
Mojmap

**Comment by moritz-htk** [comment]
The problem has nothing to do with Yarn or specifically Arch Loom. Was my mistake sorry

**Comment by Fuzss** [comment]
Issue was fixed for me by running the `clean` Gradle task (i.e. deleting the `build` directory) and regenerating run configurations (probably not necessary as well).

**Comment by ghostflyby** [comment]
maybe related to this https://github.com/McModLauncher/securejarhandler/commit/312eb116454c220a51332616a717adf0d117955f

**Comment by Jab125** [comment]
TLDR: ☢️ Gradle to fix

**Comment by forkiesassds** [comment]
> TLDR: ☢️ Gradle to fix

In my case I had to actually delete the `.idea` folder to let the run tasks to regenerate
